integration-cli/docker_cli_create_test.go

```diff
@@ -60,6 +60,27 @@ func (s *DockerSuite) TestCreateArgs(c *check.C) {
 
 }
 
+// Make sure we can grow the container's rootfs at creation time.
+func (s *DockerSuite) TestCreateGrowRootfs(c *check.C) {
+	testRequires(c, Devicemapper)
```

Comment:
Is there something missing? I can not find any code about `Devicemapper` testRequirement